bot/controllers/addGames.js

```diff
@@ -0,0 +1,30 @@
+// module.exports = () => {
+// const util = require('apex-util');
+//
+// const _run = (message) => {
+//     const disallowedRoles = ['admin', 'armada officers', 'armada officer', 'moderator', 'privateers', 'privateer', 'tester', 'crew', 'fleet officer', '@everyone'];
+//     const ctrls = [{
+//       cmd: '!addGame',
+//       example: '!addGame <game_name>',
+//       title: 'Add Game',
+//       desc: 'Add a single game to yourself. game is case-sensitive.',
```

Comment:
The context of this text seems confusing, consider "Add new game channel to server"?